documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,139 @@
+
+#### WebLogic Server Certification
+
+**Q:** Which Java EE profiles are supported/certified on Kubernetes, only Web Profile or WLS Java EE full blown?
+
+**A:** We support the full Java EE Profile.
+
+
+#### WebLogic Server Configuration
+
+**Q:** How is the WebLogic Server domain configured in a Docker container (for example, databases, JMS, and such) that is potentially shared by many domains?
+
+**A:** In a Kubernetes and Docker environment, the WebLogic domain home can be externalized to a persistent volume, or supplied in an image (by using a layer on top of a WebLogic Server image). For WebLogic domains that are supplied using an image, the domain logs and store locations optionally can be located on a persistent volume.  See the [samples]({{< relref "/samples/simple/_index.md" >}}) in this project.
+
+When using the operator, each deployed domain is specified by a domain resource that you define which describes important aspects of the domain. These include the location of the WebLogic Server image you wish to use, a unique identifier for the domain called the `domain-uid`, any PVs or PVC the domain pods will need to mount, the WebLogic clusters and servers which you want to be running, and the location of its domain home.
+
+Multiple deployments of the same domain are supported by specifying a unique `domain-uid` string for each deployed domain and specifying a different domain resource. The `domain-uid` is in turn used by the operator as the name-prefix and/or label for the domain's Kubernetes resources that the operator deploys for you. The WebLogic configuration of a domain's deployments optionally can by customized by specifying configuration overrides in the domain resource -- which, for example, is useful for overriding the configuration of a data source URL, user name, or password.
+
+The operator does not specify how a WebLogic domain home configuration is created. You can use WLST, REST, or a very convenient new tool called [WebLogic Server Deploy Tooling](https://github.com/oracle/weblogic-deploy-tooling) (WDT). WDT allows you to compactly specify WebLogic configuration and deployments (including JMS, data sources, applications, authenticators, and such) using a YAML file and a ZIP file (which include the binaries). The operator [samples]({{< relref "/samples/simple/_index.md" >}}) show how to create domains using WLST and using WDT.
+
+
+**Q:** Is the Administration Server required? Node Manager?
+
+**A:** Certification of both WebLogic on Docker and WebLogic in Kubernetes consists of a WebLogic domain with the Administration Server. The operator configures and runs Node Managers for you within a domain's pods - you don't need to configure them yourself - so their presence is largely transparent.
```

Comment:
In other places, we've moved away from highlighting Docker to mentioning "container images" or running in "containers". Many if not most will understand that Docker support means that we also support other container runtimes, such as cri-o, but we make this clearer by no longer preferencing Docker.

Reply:
I'll change it.

Reply:
Done. You'll see it in the next commit.